### DISCUSSION TITLE
About shadowsocks 2023?

### DISCUSSION THREAD

**Discussion by moranno** [database64128/shadowsocks-go#42] [General]
I think I read somewhere a few months ago that you discussed improvements related to ss2022, and you seemed to mention that these improvements would be a proposal for ss2023.

Could you please talk more about ss2023 and related improvement proposals? 
I can't find the previous discussion. Thanks!

@database64128

**Comment by database64128** [comment]
There are no public discussions on Shadowsocks 2023 at the moment. Hopefully I'll have more to share in May. For now, here's a list of goals and potential features:

- Perfect Forward Secrecy with X25519 ECDH.
- Optional 0-RTT initial payload w/o PFS guarantee.
- First-class multi-user support.
- UoT and multiplexing.

**Reply by moranno** [reply]
Could you please talk more about it now?
